projects/organization-management/src/app/components/user-budget/user-budget.component.html

```diff
@@ -25,8 +25,9 @@
           [ngClass]="{ 'bg-danger': usedBudgetPercentage * 100 > 90 }"
           [ngStyle]="{ width: usedBudgetPercentage | percent }"
           role="progressbar"
+          aria-labelledby="progessbar-percentage"
         >
-          <span class="progress-display">{{ usedBudgetPercentage | percent }}</span>
+          <span id="progessbar-percentage" class="progress-display">{{ usedBudgetPercentage | percent }}</span>
```

Comment:
this component is used several times at once on the users list. So this way the id won't be unique on page.

Reply:
decided to remove the `id` and changed to `aria-hidden=true`